### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by angel-mora**
second microverse project completed and reviewed, ready to production

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                    <p>One account for everything Intuit, including Mint.</p>
+                    <p class="learn-more">Learn more</p>
+                </div>
+                <div class="form-input">
+                    <label for="email">Email address</label>
+                    <input id="email" type="email" name="email">
+                    <div></div>
+
+                    <label for="phone">Phone (recommended)</label>
+                    <input id="phone" type="text" name="phone">
```
Comment (kobiyoyo) [inline index.html]: I recommend using **tel** for input type

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team   
#### Great Job so far, but there are few more things required. 

### Page structure
- [x] Please let's add linters to your project.


- [x] Files not related to the project should not be committed and added to .gitignore (e.g. .DS_Store)

- [x] Please let's fix these Position errors for the icons (under intuit logo) and  the padlock
Your Version


Original Version



- [x] The Phone `<input>` type shouldn't be `text` 


[OPTIONAL] -  [`<nav>` tag](http://html5doctor.com/nav-element/) should be used for  major navigation links
[OPTIONAL] - [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) should be used for dominant content of the `<body>` of a document

### Make changes and submit another code review request.
*Happy coding*

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Your project looks :+1:  good!
### Great job thus far. 

### But it needs a few changes before can be approved.

#### General
* Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history  screen


### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

*Happy coding 😄*

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
The project needs changes

File: index.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <!-- font awesome -->
```
Comment (Anna-Myzukina) [inline index.html]: It’s better to delete commented code. If you need to review a previous version of your code, use git history

File: index.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <!-- font awesome -->
+    <script src="https://kit.fontawesome.com/833e2db6e5.js" crossorigin="anonymous"></script>
+
+    <!-- Google Font Montserrat Prompt -->
```
Comment (Anna-Myzukina) [inline index.html]: It’s better to delete commented code. If you need to review a previous version of your code, use git history

File: index.html [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:300Prompt&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="main.css">
+    <title>Html Forms</title>
+</head>
+
+<!-- cp am 1 -->
+
```
Comment (Anna-Myzukina) [inline index.html]: It’s better to delete commented code. If you need to review a previous version of your code, use git history

File: index.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+                <button type="submit" class="sign-in-btn"><span class="sign-in-lock-icon"></span>
+
+                    Sign in </button>
+            </div>
+        </nav>
+    </header>
+
+    <!-- cp ak 2 -->
+
+    <!-- Logo Section -->
```
Comment (Anna-Myzukina) [inline index.html]: It’s better to delete commented code. If you need to review a previous version of your code, use git history

File: index.html [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    <div class="logos-section">
+        <div class="logo"></div>
+    </div>
+
+    <!-- Three Icon section -->
```
Comment (Anna-Myzukina) [inline index.html]: It’s better to delete commented code. If you need to review a previous version of your code, use git history

File: index.html [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+                    <input id="phone" type="tel" name="phone">
+                    <p>Standard call, messaging or data rates may apply</p>
+                    <div></div>
+
+                    <label for="password">Password</label>
+                    <input id="password" type="password" name="password">
+                    <!-- <img src="assets/lock-left-grey.png"
+                        alt="open lock"> -->
+
+                    <!-- <span class="lock"></span> -->
```
Comment (Anna-Myzukina) [inline index.html]: It’s better to delete commented code. If you need to review a previous version of your code, use git history

File: index.html [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+                    <div class="form-footer">
+                        <p>Invisible reCAPTCHA by Google <span>Privacy Policy</span> and <span>Terms of Use.</span></p>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </main>
+
+    <!-- cp am 2 -->
+
```
Comment (Anna-Myzukina) [inline index.html]: It’s better to delete commented code. If you need to review a previous version of your code, use git history

File: main.css [hunk 9]
```diff
@@ -0,0 +1,230 @@
...
+  display: inline-block;
+  height: 1em;
+  margin-top: 2px;
+  margin-right: 4px;
+  vertical-align: text-top;
+  width: 0.8em;
+}
+
+/* cp ak 2 */
+
```
Comment (Anna-Myzukina) [inline main.css]: It’s better to delete commented code. If you need to review a previous version of your code, use git history

File: main.css [hunk 10]
```diff
@@ -0,0 +1,230 @@
...
+  background-image: url(https://plugin.intuitcdn.net/identity-authn-core-ui/images/ecosystem_logos_new@2x-a8356d8c.png);
+  background-size: 562px 83px;
+  margin: auto;
+  height: 35px;
+  width: 118px;
+}
+
+.logo-icons-section {
+  /* margin: auto;
+    position: fixed; */
```
Comment (Anna-Myzukina) [inline main.css]: It’s better to delete commented code. If you need to review a previous version of your code, use git history

File: main.css [hunk 11]
```diff
@@ -0,0 +1,230 @@
...
+
+.form-input input {
+  align-self: center;
+  height: 34px;
+  padding: 0 6px 0 8px;
+  width: 320px;
+}
+
+/* cp am 3 */
+
```
Comment (Anna-Myzukina) [inline main.css]: It’s better to delete commented code. If you need to review a previous version of your code, use git history

File: main.css [hunk 12]
```diff
@@ -0,0 +1,230 @@
...
+  color: #757575;
+  padding: 2px;
+}
+
+span {
+  color: #0077c5;
+}
+
+/* cp ak 4 */
+
```
Comment (Anna-Myzukina) [inline main.css]: It’s better to delete commented code. If you need to review a previous version of your code, use git history

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Comment by addod19** [comment]
## Pull Request under reiview by Daniel Larbi Addo (TSE)

File: index.html [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+        </div>
+    </div>
+
+    <main class="form-section">
+        <div class="form-body">
+            <form action="#" method="post">
+                <div class="form-header">
+                    <h2 class="sub-title">Create an Intuit account</h2>
+                    <p>One account for everything Intuit, including Mint.</p>
+                    <p class="learn-more">Learn more</p>
```
Comment (addod19) [inline index.html]: Kindly add a link to ```<p class="learn-more">Learn more</p>```

**Review by addod19 — APPROVED** [review]
## Status: Approved 🟢

Hi Team, great job implementing all the changes requested by previous reviewers. 👍 
Kindly add a link to the ```Learn more``` paragraph. Before you merge to master.
**Congratulations** on a great job ✋ 

Happy coding 💻 
Good luck in your next project. 😄

**Pull request merged**